Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "DataStructures"
 uuid = "864edb3b-99cc-5e75-8d2d-829cb0a9cfe8"
-version = "0.17.10"
+version = "0.17.11"
```

Comment:
I am unclear as to if this is a breaking change.
I don't *think* we expost this as part of our public API, but I am not sure.
Reviewer should take note and consider carefully.